src/components/navigation/SideBar.tsx

```diff
@@ -116,7 +116,7 @@ const SideBar = (): JSX.Element => {
 				</VStack>
 				{isDrawerNeeded && (
 					<ProfileBadge
-						username={user ? user.contact.username : 'IPC'}
+						username={user ? user.fullContact.contact.username : 'IPC'}
 						address={(user ? user.account.address : 'IPC') || 'IPC'}
```

Comment:
```suggestion
						username={user?.fullContact.contact.username || 'IPC'}
						address={user?.account.address || 'IPC'}
```
looks better like this (make sure it works)